datadog/resource_datadog_logs_pipeline.go

```diff
@@ -0,0 +1,725 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/zorkian/go-datadog-api"
+	"strings"
+)
+
+const (
```

Comment:
Can we group remappers and processors please ? It should ease readability

Reply:
I was sorting them alphabetically. It should be easier for the people who are not familiar with logs backend related logic? In addition, they are all called `processors`.

Reply:
I don’t think so, it just grouping by suffix basically.

Reply:
```
const (
	tfArithmeticProcessor        = "arithmetic_processor"
	tfAttributeRemapperProcessor = "attribute_remapper"
	tfCategoryProcessor          = "category_processor"
	tfDateRemapperProcessor      = "date_remapper"
	tfGrokParserProcessor        = "grok_parser"
	tfMessageRemapperProcessor   = "message_remapper"
	tfNestedPipelineProcessor    = "pipeline"
	tfServiceRemapperProcessor   = "service_remapper"
	tfStatusRemapperProcessor    = "status_remapper"
	tfTraceIdRemapperProcessor   = "trace_id_remapper"
	tfUrlParserProcessor         = "url_parser"
	tfUserAgentParserProcessor   = "user_agent_parser"
)
```
To me the order is alphabetic:
```
tfArithmeticProcessor        
tfAttributeRemapperProcessor 
tfCategoryProcessor          
tfDateRemapperProcessor      
tfGrokParserProcessor        
tfMessageRemapperProcessor   
tfNestedPipelineProcessor    
tfServiceRemapperProcessor   
tfStatusRemapperProcessor    
tfTraceIdRemapperProcessor   
tfUrlParserProcessor         
tfUserAgentParserProcessor
```